pandas/io/wb.py

```diff
@@ -1,314 +1,6 @@
-    out.columns = ['country', 'iso_code', 'year', indicator]
-    return out,"Success"
-
-def get_countries():
-    """Query information about countries
-    """
-    url = 'http://api.worldbank.org/countries/?per_page=1000&format=json'
-    with urlopen(url) as response:
-        data = response.read()
-    data = json.loads(data)[1]
-    data = pandas.DataFrame(data)
-    data.adminregion = [x['value'] for x in data.adminregion]
-    data.incomeLevel = [x['value'] for x in data.incomeLevel]
-    data.lendingType = [x['value'] for x in data.lendingType]
-    data.region = [x['value'] for x in data.region]
-    data = data.rename(columns={'id': 'iso3c', 'iso2Code': 'iso2c'})
-    return data
-
-def get_indicators():
-    """Download information about all World Bank data series
-    """
-    url = 'http://api.worldbank.org/indicators?per_page=50000&format=json'
-    with urlopen(url) as response:
-        data = response.read()
-    data = json.loads(data)[1]
-    data = pandas.DataFrame(data)
-    # Clean fields
-    data.source = [x['value'] for x in data.source]
-    fun = lambda x: x.encode('ascii', 'ignore')
-    data.sourceOrganization = data.sourceOrganization.apply(fun)
-    # Clean topic field
-
-    def get_value(x):
-        try:
-            return x['value']
-        except:
-            return ''
-    fun = lambda x: [get_value(y) for y in x]
-    data.topics = data.topics.apply(fun)
-    data.topics = data.topics.apply(lambda x: ' ; '.join(x))
-    # Clean outpu
-    data = data.sort(columns='id')
-    data.index = pandas.Index(lrange(data.shape[0]))
-    return data
-
-_cached_series = None
-
-
-def search(string='gdp.*capi', field='name', case=False):
-    """
-    Search available data series from the world bank
-
-    Parameters
-    ----------
-
-    string: string
-        regular expression
-    field: string
-        id, name, source, sourceNote, sourceOrganization, topics
-        See notes below
-    case: bool
-        case sensitive search?
-
-    Notes
-    -----
-
-    The first time this function is run it will download and cache the full
-    list of available series. Depending on the speed of your network
-    connection, this can take time. Subsequent searches will use the cached
-    copy, so they should be much faster.
-
-    id : Data series indicator (for use with the ``indicator`` argument of
-    ``WDI()``) e.g. NY.GNS.ICTR.GN.ZS"
-    name: Short description of the data series
-    source: Data collection project
-    sourceOrganization: Data collection organization
-    note:
-    sourceNote:
-    topics:
-    """
-    # Create cached list of series if it does not exist
-    global _cached_series
-    if type(_cached_series) is not pandas.core.frame.DataFrame:
-        _cached_series = get_indicators()
-    data = _cached_series[field]
-    idx = data.str.contains(string, case=case)
-    out = _cached_series.ix[idx].dropna()
-    return out
+raise ImportError(
```

Comment:
It doesn't is much effort to keep it .. ? (and it's for the same reason as the import error in data.py), but indeed probably used a lot less